compliance/long_running_isntances/README.md

```diff
@@ -0,0 +1,25 @@
+## Long Running Instances 
+
+
+### What it does
+
+This policy template checks all instances running, and checks them against a user provided number of days.  If the instance has been running longer than the specified number of days then the policy reports those instances for termination, and seeks approval before termination.  The policy accepts using tags to not include instances.
```

Comment:
change "The policy accepts using tags to not include instances." to "The policy accepts using tags to exclude instances."